### ISSUE TITLE
tests/drivers/spi/spi_loopback/peripheral.spi fails to build on several boards

### ISSUE THREAD

**Issue by galak** [zephyrproject-rtos/zephyr#15495]
```
particle_xenon            tests/drivers/spi/spi_loopback/peripheral.spi      FAILED: build_error
	see: sanity-out/particle_xenon/tests/drivers/spi/spi_loopback/peripheral.spi/build.log

total complete:    2/  13  15%  failed:    2

particle_boron            tests/drivers/spi/spi_loopback/peripheral.spi      FAILED: build_error
	see: sanity-out/particle_boron/tests/drivers/spi/spi_loopback/peripheral.spi/build.log

total complete:    3/  13  23%  failed:    3

particle_argon            tests/drivers/spi/spi_loopback/peripheral.spi      FAILED: build_error
	see: sanity-out/particle_argon/tests/drivers/spi/spi_loopback/peripheral.spi/build.log

total complete:    7/  13  53%  failed:    7
```

**Comment by pabigot** [comment]
There's a fix for this in #15369 if somebody wants to cherry-pick and merge it.

**Comment by galak** [comment]
> There's a fix for this in #15369 if somebody wants to cherry-pick and merge it.

Thanks, I'll cherry-pick that out of the PR for 1.14.

**Comment by galak** [comment]
Resolved by #15496
